Makefile

```diff
@@ -261,7 +261,7 @@ test: $(test-targets)
 unit-test: ## Runs unit test without e2e
 	# Run unit tests, ignoring e2e tests
 	# run the go tests and gen the file coverage-all used to do the integration with codecov
-	go test -race -covermode=atomic  -coverprofile=unit-coverage.out `go list ./... | grep -v e2e`
+	SKIP_GINKGO=1 go test -race -covermode=atomic  -coverprofile=unit-coverage.out `go list ./...`
```

Comment:
Is there a reason these e2e tests couldn't sit in the e2e dir?

Reply:
@06kellyjac  Nice suggestion, Thanks. Do you want to do a PR for the change?

Reply:
The tests added in this PR rely on some internal (non-exposed) fields. So, had to create the tests in the same package as it was testing.

Reply:
fair enough, thanks for the details